bff/test/Duende.Bff.Tests/TestFramework/TestLoggerProvider.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Duende.Bff.Tests.TestFramework
 {
-    public class TestLoggerProvider : ILoggerProvider
+    public class TestLoggerProvider(WriteTestOutput? writeOutput = null, string? name = null) : ILoggerProvider
```

Comment:
Can we fix this nullability compiler complain please?

Reply:
I'll do a pass to enable nullable everywhere in a follow up pr, if that's ok with you? for now, the code is correct but the project nullability settings are not.